noxfile.py

```diff
@@ -216,7 +258,7 @@ def tests(session: Session) -> None:
         session.notify("coverage")
 
 
-@nox.session
+@nox_session
```

Comment:
we could skip the cast, if we used `@nox_session()`, which nox.session is missing a type for